packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -170,6 +170,7 @@ const PipelineWrapper: React.FC<IProps> = ({
   const [alert, setAlert] = React.useState('');
 
   const pipelineRuntimeName = pipeline?.pipelines?.[0]?.app_data?.runtime;
+  const pipelineRuntimeType = pipeline?.pipelines?.[0]?.app_data?.runtime_type;
```

Comment:
I'm not sure we need to add the runtime type to the pipeline if we can always derive it from the runtime instance or its schema. Look into pulling this from schema or instance.